### DISCUSSION TITLE
How can I change the default link at the top left corner? (John Doe link)

### DISCUSSION THREAD

**Discussion by noloman** [luizdepra/hugo-coder#906] [Q&A]
Hi all,

I'm trying to find out how to change the link at the top-left of the website, where the JOHNDOE text is, which points to `localhost`, but I can't seem to find it. 
I'm looking for the text "https://hugo-coder.netlify.app" sinde that's where the demo website points to, but the only reference I can find to is a `theme.toml` with `demosite = "https://hugo-coder.netlify.app/"`. Thanks!

**Comment by luizdepra** [comment]
Hey!

Well, the link in the title points to the configured `baseURL`. It uses the absolute path to it. See [here](https://github.com/luizdepra/hugo-coder/blob/main/layouts/partials/header.html#L4). So, if you are running locally with `hugo server` it will link to localhost. And, if you generate the site with the `hugo` command, it will point to the `baseURL` configured at your `config.toml`.

**Reply by noloman** [reply]
So it's absLangUrl? I've tried to override it and also the baseURL and it still points to local host when it's deployed to Netlify

**Reply by luizdepra** [reply]
I'm deploying my own site at Netlify. So, take a look [here](https://github.com/luizdepra/luizdepra.dev/blob/main/netlify.toml#L3).
I'm passing the `baseUrl` via command line, so it is using a configured domain name, set in Netlify's DNS service.
